homeassistant/components/energyid/strings.json

```
@@ -0,0 +1,66 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Connect to EnergyID",
+        "description": "Enter your EnergyID webhook provisioning key and secret. Find these in your EnergyID integration setup under provisioning credentials.\n\nMore info: {docs_url}",
+        "data": {
+          "provisioning_key": "Provisioning key",
+          "provisioning_secret": "Provisioning secret"
+        },
+        "data_description": {
+          "provisioning_key": "Your unique key for provisioning.",
+          "provisioning_secret": "Your secret associated with the provisioning key."
+        }
+      },
+      "auth_and_claim": {
+        "title": "Claim device in EnergyID",
+        "description": "This Home Assistant connection needs to be claimed in your EnergyID portal before it can send data.\n\n1. Go to: {claim_url}\n2. Enter code: **{claim_code}**\n3. (Code expires: {valid_until})\n\nAfter successfully claiming the device in EnergyID, click **Submit** below to continue."
+      },
+      "reauth_confirm": {
+        "title": "Reauthenticate EnergyID",
+        "description": "Please re-enter your EnergyID provisioning key and secret to restore the connection.\n\nMore info: {docs_url}",
+        "data": {
+          "provisioning_key": "Provisioning key",
+          "provisioning_secret": "Provisioning secret"
+        },
+        "data_description": {
+          "provisioning_key": "Your unique key for provisioning.",
+          "provisioning_secret": "Your secret associated with the provisioning key."
```

These should be deduplicated by using references instead:

```suggestion
          "provisioning_key": "[%key:component::energyid::config::step::user::data::provisioning_key%]",
          "provisioning_secret": "[%key:component::energyid::config::step::user::data::provisioning_secret%"
        },
        "data_description": {
          "provisioning_key": "[%key:component::energyid::config::step::user::data_description::provisioning_key%]",
          "provisioning_secret": "[%key:component::energyid::config::step::user::data_description::provisioning_secret%"``